### DISCUSSION TITLE
Usage of `sel` when restarting from previous frozen model

### DISCUSSION THREAD

**Discussion by francescomambretti** [deepmodeling/deepmd-kit#3935] [Q&A]
Dear all, I have a question concerning the usage of `sel` parameter. Let's say that I have a model 1, as a result of training on a given dataset. This model has a given value for `sel`. Let us suppose that I want to train a model 2, starting from the frozen model 1 using the keyword `--init-frz-model`, where the dataset now contains a new set of configurations which require a different (larger) value of `sel`, at least for some chemical elements of the system. Is it possible to use a new value for `sel` while restarting from model 1, or do I have to train from scratch on the new dataset (containing all the configurations) using the larger `sel`? Thanks!

**Comment by wanghan-iapcm** [comment]
Currently it is not supported. I have transferred the discussion to issue #3939.
